### PR TITLE
fix: Correcting the routes for my-session

### LINKED ISSUE #3594: Speaker Editing Session: Results partly seeing admin menu
If a speaker accesses talks through "My Sessions" (https://eventyay.com/my-sessions) and tries to edit his/her proposal the speaker sees the admin menu accesses links as the following example:
* https://eventyay.com/events/950/speakers/edit/4157
* https://eventyay.com/events/950/sessions/edit/5594
After saving the speaker sees the admin menu of the event and has access to the scheduler.

Expected:
* Accessing talks through the link should be the same as when the speaker tries to edit the talks or speaker details through the actual call for speaker website, e.g.:
https://eventyay.com/e/a74665c3/cfs/edit/4258 (speaker edit)
https://eventyay.com/e/a74665c3/cfs/edit/5642 (session edit)
Best would be to just use exactly the same links.
* No access to the scheduler should be possible for a standard speaker.

Screenshot from 2019-11-03 18-28-12

### PR DESCRIPTION
**Pull request by kushthedude**
Fixes #3594

### CODE REVIEW

**Comment by kushthedude** [comment]
@mariobehling @iamareebjamal Please Review & Merge

**Comment by kushthedude** [comment]
@iamareebjamal All the tests related to `Login` are failing again, I have restarted the build for development branch if it doesn't correct up. I will open an issue and debug it.
PS: The following failure has no related to this PR, Hence it's safe to merge.

**Comment by kushthedude** [comment]
Finally the Green Ticks ❤️ , @iamareebjamal Please review & merge

**Comment by iamareebjamal** [comment]
Steps taken to fix it?

**Comment by kushthedude** [comment]
> Steps taken to fix it?

Creation of test account which was removed in db

**Comment by iamareebjamal** [comment]
Automate that in the tests please. Create an issue and a PR

**Comment by kushthedude** [comment]
What about the solution I told in gitter?

**Comment by iamareebjamal** [comment]
No, that is overkill

**Comment by kushthedude** [comment]
What should be the solutions , create an account and delete it after and
then create a new in script ?

**Comment by iamareebjamal** [comment]
No, try to login, and if it fails, create a new account

**Comment by kushthedude** [comment]
@iamareebjamal we can atleast merge this and make the release for now.

**Comment by iamareebjamal** [comment]
We can merge this but release will be made after event wizard fix

**Comment by iamareebjamal** [comment]
Get a review from peers

**Comment by kushthedude** [comment]
@prateekj17 !

**Comment by iamareebjamal** [comment]
@prateekj117 Update?

**Comment by prateekj117** [comment]
@iamareebjamal Back online!!! Looking into it.

File: app/controllers/my-sessions/view.js [hunk 1]
```diff
@@ -1,34 +1,38 @@
...
+      .catch(() => {
+        this.notify.error(this.l10n.t('An unexpected error has occurred.'));
+      })
+      .finally(() => {
+        this.set('isLoading', false);
+        this.set('isProposalDeleteModalOpen', false);
+      });
+  }
+}
+
```
Comment (prateekj117) [inline app/controllers/my-sessions/view.js]: @kushthedude These changes are not relevant to this issue.

Comment (prateekj117) [reply app/controllers/my-sessions/view.js]: @iamareebjamal Just the changes in view.hbs are required. Changes done by @kushthedude LGTM.

**Comment by kushthedude** [comment]
We are using ES6 syntax its good to migrate.

**Comment by iamareebjamal** [comment]
It's good to migrate in a separate PR. Changes should be relevant to the issue at hand. Refactoring should be done in separate PRs to ensure no side effect bugs are introduced and git blame remains indicative of respective changes.

**Comment by iamareebjamal** [comment]
https://github.com/fossasia/open-event-server/pull/6581#discussion_r344177574

**Comment by kushthedude** [comment]
> It's good to migrate in a separate PR. Changes should be relevant to the issue at hand. Refactoring should be done in separate PRs to ensure no side effect bugs are introduced and git blame remains indicative of respective changes.

Sure, Will make revert this changes 👍

**Comment by kushthedude** [comment]
@iamareebjamal Reverted the change

File: app/controllers/my-sessions/view.js [hunk 2]
```diff
@@ -31,4 +31,4 @@ export default Controller.extend({
         });
     }
   }
-});
+});
```
Comment (iamareebjamal) [inline app/controllers/my-sessions/view.js]:
```suggestion
});

```

**Comment by kushthedude** [comment]
@iamareebjamal done, Please review.

**Pull request merged**